server/index.js

```diff
@@ -10,10 +10,12 @@ var corsHeaders = require('hapi-cors-headers')
 var hoodieServer = require('@hoodie/server').register
 var log = require('npmlog')
 var PouchDB = require('pouchdb-core')
+var cloneDeep = require('lodash').cloneDeep
```

Comment:
this is the full lodash. the alternative would have been be `require('lodash/cloneDeep')`